exercises/Loops-stringr-R.md

```diff
@@ -6,49 +6,36 @@ language: R
 ---
 
 A colleague has produced a file with one DNA sequence on each line. Download
-[the file]({{ site.baseurl }}/data/dna-sequences-1.txt) and load it into R using 
-`read.csv()`. The file has no header. Name the resulting data frame `sequences`.
+[the file]({{ site.baseurl }}/data/dna-sequences-1.txt). Load it into R with`read.table()`, setting `header = TRUE` and naming the resulting data frame `sequences`.
```

Comment:
Any reason not to use `read.csv()` which they are used to and assumes the existence of the header?

Reply:
Duh.